thirdparty/doctest/doctest.h

```diff
@@ -2868,9 +2872,11 @@ namespace detail {
             oss << std::setw(2) << static_cast<unsigned>(bytes[i]);
         return oss.str().c_str();
     }
-
-    DOCTEST_THREAD_LOCAL std::ostringstream g_oss; // NOLINT(cert-err58-cpp)
-
+	#if defined(DOCTEST_PLATFORM_WINDOWS)
```

Comment:
Let's try to keep the indentation correct even if it's a hotfix :)
And see @bruvzg's comment about editing third-party libs.

Reply:
will do sorry MSVC keeps lagging out when I type in it... I will update with a real text editor and squash at the end once we are certain this is gone :D